tests/after_cluster_deploy_sanity/test_after_cluster_deploy_sanity.py

```diff
@@ -21,6 +21,9 @@
 LOGGER = logging.getLogger(__name__)
 
 
+pytestmark = [pytest.mark.x86_vanilla]
```

Comment:
```suggestion
pytestmark = [pytest.mark.x86_default_setup]
```
ditto for all other marker additions